src/coreclr/tools/Common/TypeSystem/Common/MetadataType.cs

```diff
@@ -110,7 +110,6 @@ public struct ClassLayoutMetadata
     {
         public int PackingSize;
         public int Size;
-        public FieldAndOffset[] Offsets;
     }
 
     public struct FieldAndOffset
```

Comment:
This should be moved to `FieldLayoutAlgorithm.cs` or `FieldDesc.FieldLayout.cs` since it is no longer used in the common MetadataType.